gnd/src/main/java/com/google/android/gnd/ui/offlinearea/OfflineAreaListAdapter.java

```diff
@@ -37,6 +50,7 @@
     public int position;
     private ImmutableList<OfflineArea> areas;
     private final Navigator navigator;
+    private final Geocoder geocoder;
```

Comment:
Following our current architecture, remote and system services are all hidden either behind a `Repository` or a `Manager`, respectively. Also,
`ListAdapter` is a purely UI component, so it would be best to abstract services like this one so they can be mocked or replaced without needing to mock the lower level API.

How about creating a `GeocodingService` that exposes only the functionality needed to implement this feature?

Reply:
Done! 81c0b3f added a `GeocodingManager`